source/fundamentals/time-series.txt

```diff
@@ -0,0 +1,55 @@
+===========
+Time Series
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecols
+
+
+Overview
+--------
+
+In this guide, you can learn about time series collections in the MongoDB
+Node.js driver. 
+
+Create a Time Series Collection
+-------------------------------
+
+You should create a time series collection using the MongoDB Shell. For detailed
+instructions on creating a time series collection using the MongoDB Shell, see our
+:manual:`MongoDB Manual entry on time series collections </core/timeseries-collections/#create-a-time-series-collection>`.
+
+.. important:: Use the MongoDB Shell For Administrative Commands
+
+   You should not use the Node.js driver to perform administrative commands on
+   your MongoDB deployment. Use the MongoDB Shell to perform administrative
+   commands including but not limited to:
+   
+   - :manual:`Create a collection </reference/method/db.createCollection/#mongodb-method-db.createCollection>`
+   - :manual:`Create a user </reference/method/db.createUser/#mongodb-method-db.createUser>`
+   - :manual:`Manage access </reference/method/db.grantRolesToUser/#mongodb-method-db.grantRolesToUser>`
+
+Query a Time Series Collection
+------------------------------
+
+You query a time series collection in the same way you query other
+collection types in MongoDB. The Node.js driver has no features specifically for
+querying time series data.
```

Comment:
**Suggestion:** The second sentence seems a little overly negative.
```suggestion
While the Node.js driver has no features specifically for
querying time series data, you can query a time series 
collection in the same way you query other
collection types in MongoDB.
```

Reply:
I agree the proposed rephrasing is less harsh, but I think it may be a little less clear. I tried to think of a way to clearly communicate the following things in a less harsh way and I haven't been able to do it:

- Time Series Collections behave like normal collections
- There is no feature for querying time series collections as there is no need for this feature.

I guess I see the harshness as stopping an incorrect course of reasoning more so than criticizing the driver (stop the search for a feature that does not exist), so I am comfortable keeping the current language. Open to further discussion if you disagree!

Reply:
Perhaps "Since you query a time series collection in the same way you query other collection types in MongoDB, the Node.js driver has no features specifically for querying time series data." The current wording makes it seem like something is missing.

Reply:
I like that! Great rephrase. I will replace what is currently there with your edit.